metrics/grafana/tidb_summary.jsonnet

```
@@ -0,0 +1,436 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+
+local myDS = '${DS_TEST-CLUSTER}';
+
+// A new dashboard
+local newDash = dashboard.new(
+  title='Test-Cluster-TiDB-Summary',
+  editable=true,
+  graphTooltip='shared_crosshair',
+  refresh='30s',
+  time_from='now-1h',
+);
```

I suggest add the `__input` here, so that the generated .json can be directly imported which makes development and debugging easier.
```suggestion
).addInput(
  name="DS_TEST-CLUSTER",
  label="test-cluster",
  type="datasource",
  pluginId="prometheus",
  pluginName="Prometheus"
);
```